### PR TITLE
Add minor UI fixes 

### PR DESCRIPTION
**Pull request by ramyabuva**
This PR fixes the table rendering for plan, and moves the calendar so that adding additional courses does not move it down.

### CODE REVIEW

**Review by savsa — APPROVED** [review]

File: src/main/webapp/js/lib/util.js [hunk 1]
```diff
@@ -55,6 +56,21 @@ function createPlanTable(tableData, creditsData, courseContainer) {
   courseContainer.appendChild(table);
 }
 
+/**
+ * Rectangularizes matrix by ensuring each row has an equal number of elements.
+ * @param {Object} data The 2d array to rectangularize
+ */
+function rectangularize2dMatrix(data) {
+  const longestRow =
```
Comment (wli) [inline src/main/webapp/js/lib/util.js]: why can't this be:

`const longestRow = Math.max(data.map(arr => arr.length));`

**Review by wli — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/lib/util.js (hunk 1 above)
Comment (ramyabuva) [reply src/main/webapp/js/lib/util.js]: I tried that first, it results in NaN for some reason.

Comment (wli) [reply src/main/webapp/js/lib/util.js]: Ah, you need: `Math.max(...data.map(arr => arr.length));`

Math.max takes in a variable number of arguments, not an array.

**Review by wli — CHANGES_REQUESTED** [review]

File: src/main/webapp/js/lib/util.js (hunk 1 above)
Comment (ramyabuva) [reply src/main/webapp/js/lib/util.js]: that works!

File: src/main/webapp/js/pages/planner.js [hunk 2]
```diff
@@ -92,6 +92,11 @@ async function savePlan(courseList) {
       method: 'POST',
       body: JSON.stringify(data),
     });
+    document.getElementById('save-button-text').innerText = 'Saved!';
+    document.getElementById('save-plan-prompt')
+        .classList.remove('btn-secondary');
+    document.getElementById('save-plan-prompt').classList.add('btn-primary');
```
Comment (wli) [inline src/main/webapp/js/pages/planner.js:98]: question - is the user allowed to save again? Should this button be disabled instead?

Comment (ramyabuva) [reply src/main/webapp/js/pages/planner.js:98]: I decided  not to disable it so they could save it again under a different name if they wanted.

Comment (wli) [reply src/main/webapp/js/pages/planner.js:98]: sounds good.

**Review by wli — APPROVED** [review]

**Review by triplehair — APPROVED** [review]

**Pull request merged**